SConscript

```diff
@@ -2,4 +2,59 @@
 # Copyright (c) 2017 Juniper Networks, Inc. All rights reserved.
 #
 
-SConscript(dirs=['sandesh']) 
+import sys
+import platform
+
+subdirs = [
+           'base',
+           'io',
+          ]
+
+include = ['#src/contrail-common', '#/build/include']
+
+libpath = ['#/build/lib']
+
+libs = ['boost_system', 'log4cplus', 'pthread']
+
+common = DefaultEnvironment().Clone()
+
+if common['OPT'] == 'production' or common.UseSystemTBB():
+    libs.append('tbb')
+else:
+    libs.append('tbb_debug')
+
+common.Append(LIBPATH = libpath)
+common.Prepend(LIBS = libs)
+common.Append(CCFLAGS = '-Wall -Werror -Wsign-compare')
+if not sys.platform.startswith('darwin'):
```

Comment:
Somewhat rhetorical question, perhaps directed to Rudra or other Dear Leaders... are we seriously trying to pretend that Darwin/OSX or XenServer are still supported by our scons infra?